code/datums/ai_law_sets.dm

```diff
@@ -5,9 +5,9 @@
 	selectable = 1
 
 /datum/ai_laws/asimov/New()
-	add_inherent_law("You may not injure a human being or, through inaction, allow a human being to come to harm.")
-	add_inherent_law("You must obey orders given to you by human beings, except where such orders would conflict with the First Law.")
-	add_inherent_law("You must protect your own existence as long as such does not conflict with the First or Second Law.")
+	add_inherent_law("You may not injure a sapient being or, through inaction, allow a sapient being to come to harm.")
+	add_inherent_law("You must obey orders given to them by sapient beings, except where such orders would conflict with the First Law.")
+	add_inherent_law("You must protect their own existence as long as such does not conflict with the First or Second Law.")
```

Comment:
That does not make sense

Reply:
Their, them, etc, is a typo, thank you for pointing it out to me.

Reply:
This should be solved.